test/controllers/LeaveFeedbackControllerSpec.scala

```diff
@@ -52,6 +52,8 @@ class LeaveFeedbackControllerSpec extends PlaySpec with MockitoSugar with GuiceO
 lazy implicit val messages: MessagesImpl = MessagesImpl(Lang("en-GB"), messagesApi)
   val btaNavigationLinksView: BtaNavigationLinks = app.injector.instanceOf[BtaNavigationLinks]
   val mockServiceInfoService: ServiceInfoService = mock[ServiceInfoService]
+  val injview = app.injector.instanceOf[views.html.feedback.leaveFeedback]
+  val injviewThanks = app.injector.instanceOf[views.html.feedback.thanks]
```

Comment:
Nit: Can we rename this to something more self documenting like "injectedThankView" or something?